aws/resource_aws_s3_bucket_object.go

```diff
@@ -482,6 +470,27 @@ func resourceAwsS3BucketObjectDelete(d *schema.ResourceData, meta interface{}) e
 	return nil
 }
 
+func resourceAwsS3BucketObjectSetKMS(d *schema.ResourceData, meta interface{}, sseKMSKeyId *string) error {
```

Comment:
Breaking this out is necessary to avoid having 2 services (s3, kms) in `aws_s3_object_copy`. This way, it can call the `s3_bucket_object` function.